generated/nidaqmx/nidaqmx.proto

```diff
@@ -3241,6 +3319,39 @@ message DeleteNetworkDeviceResponse {
   int32 status = 1;
 }
 
+message DeleteSavedGlobalChanRequest {
+  string channel_name = 1;
+}
+
+message DeleteSavedGlobalChanResponse {
+  int32 status = 1;
+}
+
+message DeleteSavedScaleRequest {
+  string scale_name = 1;
+}
+
+message DeleteSavedScaleResponse {
+  int32 status = 1;
+}
+
+message DeleteSavedTaskRequest {
+  string task_name = 1;
```

Comment:
I would've expected the type on the request to mirror `SaveTaskRequest`: that being a `nidevice_grpc.Session` type to represent the task by name or id.

Reply:
Nevermind, I see now the SaveTask has a save_as field which I'm guessing what's referenced here. Makes sense.